packages/graph/hash_graph_clients/typescript/.openapi-generator/FILES

```diff
@@ -0,0 +1,10 @@
+.gitignore
```

Comment:
What is this file 🤔

Reply:
A list of all files created by the generator. I believe this is used when generating a new version as a list of files to remove